### ISSUE TITLE
compiler fatal error on lack of type arguments

### ISSUE THREAD

**Issue by theSherwood** [nim-works/nimskull#1317]
### Specification

Instead of returning information to the user about needing to specify type arguments for `Wrapper` (as in `Wrapper[int, int]`). Instead the compiler hits a fatal error.

### Example

```nim
type
  Wrapper*[K, V] = object
    node*: (K, V)

echo sizeof(Wrapper)
```

### Actual Output

```
Fatal: Internal unreachable code executed - ccgtypes.nim(708, 18) (getTypeDescAux(tyGenericParam)) should never be called.
```

### Expected Output

I would expect the compiler to fail to compile the code and print a message to the user about the lack of type arguments.

**Comment by zerbina** [comment]
As per our discussion on Matrix, I've assigned you to this issue, in order to make it easier to keep of who is working on what.

**Comment by zerbina** [comment]
Fixed by #1328.
